source/fundamentals/builders/indexes.txt

```diff
@@ -11,3 +11,165 @@ Indexes Builders
    :class: singlecol
 
 .. _indexes-builders:
+
+Overview 
+--------
+
+This guide shows you how to specify **indexes** using the Java Driver
+:doc:`builders </fundamentals/builders/>`. Index builders provide
+convenient methods for different types of indexes:
+
+- :ref:`Ascending <index-ascending>`
+- :ref:`Descending <index-descending>`
+- :ref:`Compound Indexes <index-compound-indexes>`
+- :ref:`Text Indexes <index-text-indexes>`
+- :ref:`Hashed Indexes <index-hashed-indexes>`
+- :ref:`Geospatial Indexes <index-geospatial-indexes>`
+
+Indexes store small portions of the collection’s data set. The index
+stores the value of a specific field or set of fields, ordered by the
+value of the field. See the :doc:`Indexes </fundamentals/indexes>` page
+to see examples of queries on indexes. 
+
+The :java-core-api:`Indexes <com/mongodb/client/model/Indexes.html>`
+class provides static factory methods for all the MongoDB index types.
+Each method returns a :ref:`Bson <bson>` instance, which you can pass to
+:java-sync-api:`createIndex() <com/mongodb/client/MongoCollection.html#createIndex(org.bson.conversions.Bson)>`.
+
+.. tip::
+
+   For brevity, you may choose to import all methods of the
+   :java-core-api:`Indexes <com/mongodb/client/model/Indexes.html>`
+   class statically: 
+
+   .. code-block:: java
+
+      import static com.mongodb.client.model.Indexes.*;
+
+   The examples below assume this static import.
+
+.. _index-ascending:
+
+Ascending
+---------
+
+Create a BSON instance by passing the field name or names you want to
+index to the :java-core-api:`ascending() <com/mongodb/client/model/Indexes.html#ascending(java.lang.String...)>`
+method. Then, pass the BSON instance into the createIndex() method to
+create an ascending index. 
```

Comment:
I think all the information is here, but let's reorder it so that we present what we're trying to accomplish first (show the user how to create an ascending index) instead of last. Also, make sure you distinguish between the ``Bson`` type and BSON format (here, we are talking about the former).

```suggestion
In order to create an ascending index, first call the ``ascending()`` builder method to create a ``Bson`` instance that represents the index document., passing the name or names of the fields you want to index. Then, call 
the ``createIndex()`` method on the collection, passing the``Bson`` instance that contains the index document.
```

Same recommendation for all instances.